server/pypi/packages/scipy/patches/chaquopy.patch

```diff
@@ -215,14 +289,25 @@ diff -aur src-orig/scipy/stats/_generate_pyx.py src/scipy/stats/_generate_pyx.py
      import numpy as np
      ver = tuple(int(num) for num in np.__version__.split('.')[:2])
      return ver < (1, 19)
-diff -aur src-orig/setup.py src/setup.py
---- src-orig/setup.py	2023-09-20 11:08:03.732957618 +0200
-+++ src/setup.py	2023-09-21 13:13:08.707895078 +0200
-@@ -32,6 +32,15 @@
+diff --git a/setup.py b/setup.py
+index de1c702af..85f76a66e 100755
+--- a/setup.py
++++ b/setup.py
+@@ -28,12 +28,25 @@ from tools.version_utils import write_version_py, get_version_info
+ from tools.version_utils import IS_RELEASE_BRANCH
+ import importlib
+ 
++# Chaquopy: Force use of cross-compilers for Fortran
++os.environ["FC"] = os.path.join(os.path.expanduser("~"), "Downloads/gcc-arm64-linux-x86_64/aarch64-linux-android-4.9/bin/aarch64-linux-android-gfortran")
++os.environ["F77"] = os.path.join(os.path.expanduser("~"), "Downloads/gcc-arm64-linux-x86_64/aarch64-linux-android-4.9/bin/aarch64-linux-android-gfortran")
++os.environ["F90"] = os.path.join(os.path.expanduser("~"), "Downloads/gcc-arm64-linux-x86_64/aarch64-linux-android-4.9/bin/aarch64-linux-android-gfortran")
```

Comment:
> One thing to note is that I had to hardcode some paths for Fortran in SciPy's setup.py file (despite seemingly placing the Fortran library for Android in the proper folder). 

The proper folder is not ~/Downloads, but chaquopy/server/pypi/fortran, as it says in pypi/README.md. build-wheel.py sets these variables to point into that location.